bin/configure_bind.php

```diff
@@ -62,7 +62,8 @@
 // search for dns containers
 foreach($containers->data as $container) {
     $nsIndex = 1;
-    if (isset($container->data->fields->labels->{"io.rancher.stack_service.name"})
+    if (isset($container->data->fields->dockerHostIp)
+        && isset($container->data->fields->labels->{"io.rancher.stack_service.name"})
```

Comment:
Didn't you think about setting `true` as a second parameter in `json_decode` at line 28? It would be easier write such things :)